### PR TITLE
Adds catburgers, egg with sausage, and sausage bread.

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

This is a small PR adding 3 new foods, ported from https://github.com/fulpstation/fulpstation/pull/69 with the updated sprites of https://github.com/fulpstation/fulpstation/pull/123 

Made originally by DaddyIssues and resprited by Grobelon

## Why It's Good For The Game

Egg with sausage was something I found was a shame wasn't craftable, and thought it would be something that would be liked upstream. 
I'm porting the other two foods we had as well since 'why not', if it isn't liked then I could easily remove them and just keep the egg with sausage.

image

## Changelog

:cl: DaddyIssues and Grobelon
expansion: Added 3 new foods: Catburgers, Egg with sausage, and Sausage bread.
/:cl:

### CODE REVIEW

**Comment by Zytolg** [comment]
Lizards better like-a the sausage bread.

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_burger.dm [hunk 1]
```diff
@@ -309,6 +309,17 @@
 	result = /obj/item/food/burger/empoweredburger
 	subcategory = CAT_BURGER
 
+/datum/crafting_recipe/food/catburger
+	name = "Catperson burger"
```
Comment (necromanceranne) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_burger.dm:313]: technically this can be made with normal cats too.

Comment (JohnFulpWillard) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_burger.dm]: renamed it to cat burger

**Review by Mothblocks — APPROVED** [review]
Code ✔, up to art staff now

**Pull request merged**